src/SkyApm.Diagnostics.MassTransit/Observers/ObserverSegmentContextDictionary.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SkyApm.Tracing.Segments;
+using System;
+using System.Collections.Concurrent;
+
+namespace SkyApm.Diagnostics.MassTransit.Observers
+{
+    public class ObserverSegmentContextDictionary
+    {
+        public static readonly ConcurrentDictionary<Guid, SegmentContext> Contexts = new ConcurrentDictionary<Guid, SegmentContext>();
```

Comment:
not recommended usage

Reply:
Sure, let me seperate that in each Observer to solve the Concurrent conflict.